src/engine/controls/bpmcontrol.cpp

```diff
@@ -731,53 +731,23 @@ double BpmControl::getBeatMatchPosition(
         double dThisPosition, bool respectLoops, bool playing) {
     // Without a beatgrid, we don't know the phase offset.
     if (!m_pBeats) {
+        qDebug() << "############1";
```

Comment:
leftover from debugging?

Reply:
oh yes, this slipped though.

Reply:
In this case I would recommend a force push. Keeping this code in the repo doesn't make any sense at all, even if it builds.